assets/js/buttons-examples.js

```diff
@@ -59,7 +59,13 @@ $(document).ready(function () {
           label: "Gradient",
           classes: "gradient-btn",
         },
-       
+        {
+          type: "neon-border",
+          label: "Neon Border",
+          text: "BUTTON",
+          classes: "neon-border-btn",
+          shouldHaveRoundedType: false,
+        },
```

Comment:
@RakshitaTantry28 Great Work! Just two more things, Can you put the this object in alphabetical order and change the `BUTTON` to `Button`.

Reply:
done!!